bambora-client.gemspec

```
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'bambora/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'bambora-client'
+  spec.version       = Bambora::VERSION
+  spec.authors       = ['Cassidy K']
+  spec.email         = ['hello@cassidy.codes', 'tech@himama.com']
+
+  spec.summary       = 'A thread-safe client for the Bambora/Beanstream API.'
+  spec.description   = 'The official beanstream-ruby gem is not thread-safe. This thread-safe client works in '\
+                         'environments like Sidekiq and Puma.'
+  spec.homepage      = 'https://github.com/HiMamaInc/bambora'
```

Does this need to be `bambora-client`?

```suggestion
  spec.homepage      = 'https://github.com/HiMamaInc/bambora-client'
```